<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Cronometro</class>
 <widget class="QWidget" name="Cronometro">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>799</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="btnRegresar">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Black</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 139, 139);
border-color: rgb(0, 170, 0);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Alarma</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnIniciar">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>180</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Black</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(25, 170, 59);
border-color: rgb(0, 170, 0);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Iniciar</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTiempo">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>70</y>
     <width>491</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>ROG Fonts</family>
     <pointsize>50</pointsize>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 126, 14);</string>
   </property>
   <property name="text">
    <string>00:00:00:00</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDetener">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>240</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Black</family>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(255, 47, 28);</string>
   </property>
   <property name="text">
    <string>Detener</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnMarcar">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>180</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Black</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(70, 130, 180);
border-color: rgb(0, 170, 0);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Marcar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnReiniciar">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>240</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Black</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 165, 0);
border-color: rgb(0, 170, 0);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Reiniciar</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidgetMarcas">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>311</y>
     <width>651</width>
     <height>251</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe Print</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
